frontend/src/components/phones/dashboard/PhoneDashboard.module.scss

```diff
@@ -39,7 +39,9 @@
   .senders-cta {
     display: flex;
     flex-direction: row;
-    padding: $spacing-xs $spacing-sm;
+    padding: $spacing-sm;
+    position: relative;
+    left: $spacing-sm;
```

Comment:
Hmm, I'm not really following why we're using relative positioning here (and what the `left` and `position: static` elsewhere are for) - as far as I can see the "Copied!" notice will just be displayed next to the button always? Feels like all those rules could be removed?

Reply:
For the `copied-confirmation`, I just brought in the same class from the original masks page, but taking a look to prune what I can now (was sort of in a rush to get this dashboard in! lol)

Reply:
I just looked over this PR again and pruned as much of the random positioning, I just left out the `left` with a comment about why I added that in (mainly to cancel out the padding left by the item)